.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     matrix:
       win_cxx_compilervs2015vc14:
         CONFIG: win_cxx_compilervs2015vc14
-        CONDA_BLD_PATH: C:\\bld\\
+        CONDA_BLD_PATH: D:\\bld\\
```

Comment:
Please revert this change.
(We need to fix this in conda-smithy. I'll open an issue.)

Reply:
Done.